pallets/creditcoin/src/migrations/v5.rs

```diff
@@ -124,9 +146,8 @@ pub(crate) fn migrate<T: Config>() -> Weight {
 	}
 
 	let module = crate::Pallet::<T>::name().as_bytes();
-	migration::remove_storage_prefix(module, b"UnverifiedTransfers", b"");
-	migration::remove_storage_prefix(module, b"UnverifiedCollectedCoins", b"");
-
+	let _results = migration::clear_storage_prefix(module, b"UnverifiedTransfers", b"", None, None);
+	let _results = UnverifiedCollectedCoins::<T>::clear(u32::MAX, None);
```

Comment:
I would expect these two lines to achieve the same thing, why aren't we sticking with one pattern?